### PR TITLE
started pets policy and controller

### PR DESCRIPTION
**Pull request by Elena-Learning**

### CODE REVIEW

**Comment by Elena-Learning** [comment]
please have a look at pets controller + policy

File: app/controllers/pets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  def new
+    @pet = current_user.pets.new
+    authorize @pet
+  end
+
+  def create
+    @pet = current_user.pets.new(pet_params)
+    authorize @pet
+    if pet.save
```
Comment (0math) [inline app/controllers/pets_controller.rb]: Shouldn't it be @pet.save?

**Comment by albertoT75** [comment]
It looks fine to me, dd you test it?

File: app/controllers/pets_controller.rb (hunk 1 above)
Comment (albertoT75) [reply app/controllers/pets_controller.rb]: Right

Comment (Elena-Learning) [reply app/controllers/pets_controller.rb]: Fixed, thank you

File: app/controllers/pets_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class PetsController < ApplicationController
+
+  def show
+    @pet = Pet.find(params[:id])
+    authorize @pet
+  end
+
+  def new
+    @pet = current_user.pets.new
```
Comment (MarynaG) [inline app/controllers/pets_controller.rb]: Why don't you use just Pet.new?

Comment (Elena-Learning) [reply app/controllers/pets_controller.rb]: because we need the current_user

**Pull request merged**